gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -153,6 +155,15 @@ public View onCreateView(
 
     HomeScreenFragBinding binding = HomeScreenFragBinding.inflate(inflater, container, false);
     binding.featureSheetChrome.setViewModel(viewModel);
+    binding
+        .featureSheetHeader
+        .getRoot()
+        .setOnTouchListener(
+            (v, event) -> {
+              Log.e("!!!", "Touch: " + event);
+              return binding.bottomSheetScrollView.dispatchTouchEvent(event);
```

Comment:
@dturner ..but I'm having trouble transferring touch events from the header into the bottom sheet. It works when clicking or dragging on some parts of the header, but when clicking on text or the image the touch listener here isn't called. Any ideas how to capture touch events recursively?